libbeat/processors/add_cloud_metadata/add_cloud_metadata.go

```diff
@@ -31,10 +31,17 @@ const (
 
 	// Google GCE Metadata Service
 	gceMetadataURI = "/computeMetadata/v1/?recursive=true&alt=json"
+
+	// Tencent Clound Metadata Service
+	qcloudMetadataHost          = "metadata.tencentyun.com"
```

Comment:
Now that we are starting to add more cloud providers it would be nice to have each provider in its own file. This makes it easier to spot which parts are specific to a provider and which is the shared code. Just a general note, could also be done in a follow up PR.